int32 acc_cmd
